fm/src/main.js

```diff
@@ -0,0 +1,14 @@
+import { log } from "./build/log";
+import { bleh } from "./page";
+
+import version2 from "./build/build.json" assert {type: "json"}
+export const version = version2;
+export const theme_version = {
+    state: ""
+}
+
+// TODO: Hacky, fix asap!
+setTimeout(() => {
+    log(`starting ${version.build}.${version.sku}`, 'load');
+    bleh();
+}, 1000);
```

Comment:
```suggestion
log(`starting ${version.build}.${version.sku}`, 'load');
bleh();
```

at least in tampermonkey this seems to work? let me know if it doesnt in others